docs/sources/set-up/migrate/deploy-alloy-with-otel-collector.md

```diff
@@ -0,0 +1,30 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/set-up/migrate/deploy-alloy-with-otel-collector/
+description: Learn how to deploy Grafana Alloy alongside another distribution of the OpenTelemetry Collector
+menuTitle: Deploy Alloy alongside another OTel Collector
+title: Deploy Grafana Alloy alongside another OpenTelemetry Collector distribution
+weight: 500
+---
+
+# Deploy {{% param "FULL_PRODUCT_NAME" %}} alongside another OpenTelemetry Collector distribution
+
+You can deploy both {{< param "PRODUCT_NAME" >}} and another OpenTelemetry Collector distribution in your architecture.
+A common use case for doing this is when you're migrating from a third-party vendor to Grafana OSS or Grafana Cloud.
+During migration, you may need to send telemetry to both Grafana and your legacy vendor backend.
+Although {{< param "PRODUCT_NAME" >}} includes several third-party vendor exporters, these are provided as [community components][] without commercial support.
+To get commercial support for these specific OpenTelemetry components from your legacy vendor, you'll usually need to deploy them as part of the [OpenTelemetry Collector Contrib distribution][].
```

Comment:
Is this referring to commercial support from another vendor? It'd be good to make it more clear who is providing the commercial support.